rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Hello! 😊 I wanted to point out that it's important to ensure both Getters and Setters are present for `@book` and `@person` currently, you have only set up Getters for it. 🔄 To enhance the functionality of your project, consider implementing Setters as well. This will allow you to efficiently manage and update the `@book` and `@person` data as needed. If you have any questions or need assistance with this, feel free to reach out. Happy coding! 🌟🔧

Reply:
changes made accordingly